src/main/webapp/WEB-INF/tags/instructor/results/noResponsePanel.tag

```diff
@@ -27,7 +27,12 @@
                 <tbody>
                     <c:forEach items="${noResponsePanel.emails}" var="email">
                         <tr>
-                            <td>${fn:escapeXml(noResponsePanel.teams[email])}</td>
+                            <c:if test="${fn:escapeXml(noResponsePanel.teams[email])  == 'Instructors'}">
```

Comment:
Actually since `Const.java` is already imported via `<%@ tag import="teammates.common.util.Const"%>`, can we use the constant directly for comparison?

Also, maybe it's neater to conditionally apply the css class `.italic`, rather than using `<i>` tags directly.